lib/users.js

```diff
@@ -29,7 +29,7 @@ users.sendPasswordResetEmail = async (userAccount) => {
   const accountJSON = JSON.stringify(account);
   const encodedToken = Buffer.from(accountJSON).toString("base64");
 
-  await email.sendPasswordResetEmail(userAccount.email, encodedToken);
+  await email.sendPasswordResetEmail(userAccount.email, encodedToken, process.env.NODE_ENV, process.env.PORT);
```

Comment:
This is going to break production. Production environments usually use a proxy and this app runs on port xxxx, which will be passed in even though publicly the port should be 443. 

A better approach is for `sendPasswordResetEmail` to use a `HOSTNAME` process.env variable. For example, one of our engs has `ldb.hoie.kim` as their development environment.